docs/index.md

```diff
@@ -3,174 +3,270 @@
-  version = "0.0.4"
-
-  create_operator_roles = true
-  create_oidc_provider = true
-  create_account_roles = false
-
-  cluster_id = ocm_cluster_rosa_classic.rosa_sts_cluster.id
-  rh_oidc_provider_thumbprint = ocm_cluster_rosa_classic.rosa_sts_cluster.sts.thumbprint
-  rh_oidc_provider_url = ocm_cluster_rosa_classic.rosa_sts_cluster.sts.oidc_endpoint_url
-  operator_roles_properties = data.ocm_rosa_operator_roles.operator_roles.operator_iam_roles
-}
+```
+podman run --pull=always --rm registry.ci.openshift.org/ci/ocm-tf-e2e:latest cat /root/.terraform.d/plugins/terraform.local/local/ocm/1.0.1/linux_amd64/terraform-provider-ocm > ~/terraform-provider-ocm && chmod +x ~/terraform-provider-ocm
+```
 
-```
\ No newline at end of file
+You can also use specific commit images just substituting `latest` for the desired commit SHA.
+## Variables for OCM Terraform Provider
+
+The following variables that you use to create a ROSA cluster with the Terraform provider.
+
+### Required
+
+- `aws_account_id` (String) Identifier of the AWS account.
+- `cloud_region` (String) Cloud region identifier, for example 'us-east-1'.
+- `name` (String) Name of the cluster. Must be a maximum of 15 characters in length.
+
+### Optional
+
+- `autoscaling_enabled` (Boolean) Enables autoscaling.
+- `availability_zones` (List of String) availability zones
+- `aws_private_link` (Boolean) Provides private connectivity between VPCs, AWS services, and your on-premises networks, without exposing your traffic to the public internet.
+- `aws_subnet_ids` (List of String) aws subnet ids
+- `channel_group` (String) Name of the channel group from which to select the OpenShift cluster version, for example 'stable'.
+- `client_id` (String) OpenID client identifier.
+- `client_secret` (String, Sensitive) OpenID client secret.
+- `compute_machine_type` (String) Identifier of the machine type used by the compute nodes, for example `r5.xlarge`. Use the `ocm_machine_types` data source to find the possible values.
+- `default_mp_labels` (Map of String) Labels for the default machine pool. Format should be a comma-separated list of '{"key1"="value1", "key2"="value2"}'. This list will overwrite any modifications made to Node labels on an ongoing basis.
+- `destroy_timeout` (Number) Timeout in minutes for addressing cluster state in destroy resource. Default value is 60 minutes.
+- `disable_scp_checks` (Boolean) Enables you to monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics.
+- `disable_waiting_in_destroy` (Boolean) Disable addressing cluster state in the destroy resource. Default value is false
+- `disable_workload_monitoring` (Boolean) Enables you to monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics.
+- `ec2_metadata_http_tokens` (String) Which ec2 metadata mode to use for metadata service interaction options for EC2 instancescan be optional or required, available only from 4.11.0
```

Comment:
This sentence might be missing something, especially since it starts with "which". That seems strange. Also, I think "instances" and "can" need a space between them. Finally, you might what to specify that 4.11.0 is a version and add punctuation at the end.